### PR TITLE
CSS optimisations

### PR DESCRIPTION
**Pull request by mountainash**
- removed unit type on `0` values
- removed leading `0` on decimal values
- removed unused and single use var's
- value spacing consistency
- `border: none` replaced with `border: 0`
- `outline: none` replaced with `outline: 0`
- consistent tab indenting
- removed commented-out styles (they haven't been re-enabled for 2+ years)

### CODE REVIEW

File: wled00/data/simple.css [hunk 1]
```diff
@@ -180,15 +174,15 @@ button:hover {
...
+	border: 0;
+	transition: color .3s, background-color .3s;
 	font-size: 17px;
 	color: var(--c-c);
 	min-width: 44px;
 }
 
 .top button {
-	padding: var(--tbp);
+	padding: 14px 8px 10px;
```
Comment (blazoncek) [inline wled00/data/simple.css:185]: This alone is dealbreaker. Please revert the change.

File: wled00/data/simple.css [hunk 2]
```diff
@@ -492,26 +486,19 @@ input[type=range]::-moz-range-thumb {
...
 }
 
-.hd {
-	display: var(--bhd);
-}
-
 #briwrap {
 	float: right;
-	margin-top: var(--bmt);
+	margin-top: 0;
```
Comment (blazoncek) [inline wled00/data/simple.css:501]: This one as well.

File: wled00/data/simple.css [hunk 3]
```diff
@@ -19,19 +19,13 @@
...
-	--c-tbh: rgba(51, 51, 51, var(--t-b));
-	/*following are internal*/
+	--c-o: rgba(34, 34, 34, .9);
+	--c-tb : rgba(34, 34, 34, .5);
+	--c-tba: rgba(102, 102, 102, .5);
+	--c-tbh: rgba(51, 51, 51, .5);
 	--th: 70px;
 	--tp: 70px;
 	--bh: 63px;
-	--tbp: 14px 8px 10px;
```
Comment (blazoncek) [inline wled00/data/simple.css:31]: Why do you want to remove these?

**Review by blazoncek — CHANGES_REQUESTED** [review]
I welcome your effort to optimize CSS (even though linter & minimizer might do the same) but removing parametric styles is out of the question.

File: wled00/data/simple.css (hunk 1 above)
Comment (mountainash) [reply wled00/data/simple.css:185]: Happy to revert, but can you tell me the reason for this decision?

File: wled00/data/simple.css (hunk 3 above)
Comment (mountainash) [reply wled00/data/simple.css:31]: Optimisation - it's a process overhead (small) and filesize overhead (small too) for no point. It's also a developer readability overhead. These variables are only uses once. 

If they were reused multiple times - I'd have left them in.

**Comment by mountainash** [comment]
When built, the `const uint16_t PAGE_index_L` value in `html_ui.h` drops from `32012` to `31940`. In `html_simple.h` it goes down from `18069` to `18027`.

File: wled00/data/simple.css (hunk 3 above)
Comment (blazoncek) [reply wled00/data/simple.css:31]: What makes you say that? Have you combed the sources?
They get modified during runtime by JS.

Comment (mountainash) [reply wled00/data/simple.css:31]: Agghh yuk... I see that now. For the changes between light & dark mode... not nice that this is done in JS (it can/should be done solely in CSS).

Thank you 🙇🏼 for pointing that out (it's good to have a Maintainer that's aware of all the parts of the project).

I'm working on a new "look" 💅🏼 for the Web UI and will aim to remove/reduce these.

**Comment by blazoncek** [comment]
Do not be deterred by my comments or complaints.
I am in favour of updating UI but it is a **major** task since UI is a fundamental part of what WLED is.

It has to be *perfect* on first try.
